manifests/4.6/ptp-operator.v4.5.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,244 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Should the filename be v4.6.0.xxxx ?

Reply:
Yes, it should be. I forgot to change it when copying from 4.5.  Thanks @fedepaol !
Updated.